lib/game/client/game-engine.js

```diff
@@ -169,6 +192,40 @@ class GameEngine extends Game
         this.eventsManager.emit('reldens.gameEngineTabTarget', this, closerTarget, previousTarget);
     }
 
+    getScene(scene)
```

Comment:
as general rule, unless the method use more than one line and have some extra logic on it, avoid the these and just reference the property:
this.startScene(scene)
is not worth since we can just:
this.engineDriver.startScene(scene);

Reply:
Ok... coming from more structured langueges is hard to see those properties access from anywhere in the code.
I need to switch the logic ;)